updateAzaanTimers.py

```diff
@@ -12,29 +16,48 @@
 system_cron = CronTab(user='pi')
 
 now = datetime.datetime.now()
-strPlayFajrAzaanMP3Command = 'omxplayer -o local /home/pi/adhan/Adhan-fajr.mp3 > /dev/null 2>&1'
-strPlayAzaanMP3Command = 'omxplayer -o local /home/pi/adhan/Adhan-Makkah.mp3 > /dev/null 2>&1'
-strUpdateCommand = 'python /home/pi/adhan/updateAzaanTimers.py >> /home/pi/adhan/adhan.log 2>&1'
-strClearLogsCommand = 'truncate -s 0 /home/pi/adhan/adhan.log 2>&1'
+strPlayFajrAzaanMP3Command = 'omxplayer --vol 1200 -o local {}/Adhan-fajr.mp3 > /dev/null 2>&1'.format(root_dir)
+strPlayAzaanMP3Command = 'omxplayer --vol 1800 -o local {}/Adhan-Madinah.mp3 > /dev/null 2>&1'.format(root_dir)
+strUpdateCommand = 'python {}/updateAzaanTimers.py >> {}/adhan.log 2>&1'.format(root_dir, root_dir)
+strClearLogsCommand = 'truncate -s 0 {}/adhan.log 2>&1'.format(root_dir)
 strJobComment = 'rpiAdhanClockJob'
 
-#Set latitude and longitude here
-#--------------------
-lat = 42.288788
-long = -71.551678
-
-#Set calculation method, utcOffset and dst here
-#By default system timezone will be used
-#--------------------
-PT.setMethod('ISNA')
-utcOffset = -(time.timezone/3600)
-isDst = time.localtime().tm_isdst
-
-
 #HELPER FUNCTIONS
 #---------------------------------
 #---------------------------------
 #Function to add azaan time to cron
+def parseArgs():
+    parser = argparse.ArgumentParser(description='Calculate prayer times and install cronjobs to play Adhan')
+    parser.add_argument('--lat', type=float, dest='lat',
+                        help='Latitude of the location, for example 30.345621')
+    parser.add_argument('--lng', type=float, dest='lng',
+                        help='Longitude of the location, for example 60.512126')
+    parser.add_argument('--method', choices=['MWL', 'ISNA', 'Egypt', 'Makkah', 'Karachi', 'Tehran', 'Jafari'],
+                        dest='method',
+                        help='Method of calculation')
+    return parser
+
+def mergeArgs(args):
+    file_path = pathjoin(root_dir, '.settings')
+    # load values
+    lat = lng = method = None
+    try:
+        with open(file_path, 'rt') as f:
+            lat, lng, method = f.readlines()[0].split(',')
+    except:
+        print('No .settings file found')
+    # merge args
+    if args.lat:
+        lat = args.lat
+    if args.lng:
+        lng = args.lng
+    if args.method:
+        method = args.method
+    # save values
+    with open(file_path, 'wt') as f:
+        f.write('{},{},{}'.format(lat or '', lng or '', method or ''))
+    return lat or None, lng or None, method or None
```

Comment:
Subsequent runs are crashing since this returns lat and lng as a string. Cast to float before returning
```suggestion
    return float(lat) or None, float(lng) or None, method or None
```